### DISCUSSION TITLE
How can we support with marketing?

### DISCUSSION THREAD

**Discussion by Gaialegacy** [teaxyz/white-paper#125] [Q&A]
Dear community. We would like to share the whitepaper and help bring the project to more people. 
On which platforms do you want us to share them? And how do you want us to do the marketing for you?

**Comment by iman136938** [comment]
x is most likely a safe choice . try to contact some respected influencers and share your vision with them so they share it with others

**Comment by berllx** [comment]
Is there a way to participate in the translation of the documentation/website?

**Reply by berllx** [reply]
Nvm. Got it

**Comment by rav2525** [comment]
Tea channel/page is available in YouTube, LinkedIn as well. So you can support promoting Tea there as well.
You can refer there website to direct to there official channels.

**Comment by hamaadh** [comment]
I think we need do Ads on social media, specially on twitter, facebook, youtube
Discord: hamadah6595

**Comment by zeroXserdar** [comment]
You can have a look on [this topic](https://github.com/teaxyz/white-paper/discussions/224). It clearly explains how you can contribute to TEA in terms of marketing and other community related activities.
